apps/src/lib/ui/ParentLetter.jsx

```diff
@@ -0,0 +1,237 @@
+          height: 30,
+          margin: 15
+        }}
+      />
+    </header>
+  );
+};
+
+const SignInInstructions = ({loginType, sectionCode}) => {
+  let steps;
+  switch (loginType) {
+    case SectionLoginType.clever:
+      steps = (
+        <ol>
+          <li>
+            Have your students log in to their Clever account at{' '}
+            <a href="https://www.clever.com/">www.clever.com</a> (click "Sign in
+            as a student" at the top right)
+          </li>
+          <li>
+            Click on the Code.org logo on the Clever dashboard. The logo looks
+            like this:
+            <br />
+            <img
+              src="/shared/images/clever_code_org_logo.png"
+              style={{
+                width: 60,
+                margin: 10
+              }}
+            />
+          </li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.google_classroom:
+      steps = (
+        <ol>
+          <GoToSignIn />
+          <li>Choose 'Sign in with Google'</li>
+          <li>Sign in via the Google sign-in dialog</li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.picture:
+      steps = (
+        <ol>
+          <GoToSectionSignIn sectionCode={sectionCode} />
+          <li>Click on their picture password and then click 'Sign in'</li>
+          <li>
+            If your student does not remember their picture password, please
+            email me and I will provide it
+          </li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.word:
+      steps = (
+        <ol>
+          <GoToSectionSignIn sectionCode={sectionCode} />
+          <li>Type in their secret words and then click 'Sign in'</li>
+          <li>
+            If your student does not remember their password, please email me
+            and I will provide it
+          </li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.email:
+    default:
+      steps = (
+        <ol>
+          <GoToSignIn />
+          <li>
+            Have them enter their email and password and then click 'Sign in'
+          </li>
+          <li>
+            If your student does not remember their password, they can reset it
+            from the Sign in screen
+          </li>
+        </ol>
+      );
+  }
+
+  const loginTypeName = LOGIN_TYPE_NAMES[loginType];
+  return (
+    <div>
+      <p>
+        Our class uses <strong>{loginTypeName}</strong> to sign in. To have your
+        student sign in to Code.org at home, do the following:
+      </p>
+      {steps}
+    </div>
+  );
+};
+SignInInstructions.propTypes = {
+  loginType: PropTypes.oneOf(Object.values(SectionLoginType)),
+  sectionCode: PropTypes.string
+};
+
+const GoToSignIn = () => (
+  <li>
+    Go to <a href="https://studio.code.org/">studio.code.org</a> and click 'Sign
+    in'
+  </li>
+);
+
+const GoToSectionSignIn = ({sectionCode}) => {
+  const sectionUrl = `studio.code.org/sections/${sectionCode}`;
```

Comment:
usually this and other urls would be relative using urlHelpers, but doesn't seem that important right now.

Reply:
:man_facepalming: Oh, we should totally do that.  Thanks for the reminder!